libres/lib/analysis/update.cpp

```diff
@@ -523,7 +468,11 @@ update_data_type make_update_data(enkf_fs_type *source_fs,
     obs_data_type *obs_data = obs_data_alloc(global_std_scaling);
     meas_data_type *meas_data = meas_data_alloc(ens_mask);
 
-    int_vector_type *ens_active_list = bool_vector_to_active_list(ens_mask);
+    int_vector_type *ens_active_list = int_vector_alloc(0, 0);
+    for (int i = 0; i < ens_mask.size(); i++) {
+        if (ens_mask[i])
+            int_vector_append(ens_active_list, i);
+    }
```

Comment:
I guess this is fine to do, but it is what `bool_vector_to_active_list` does already

Reply:
Ignore my comment, did not read far enough.

Reply:
yes, I have a cleanup branch which removes this, so I will just leave it for now

Reply:
actually not exactly... The bool_vector_to_active_list creates an std::vector after this PR